### PR TITLE
chore(blade): prevent internal story removal on prod

### PR DESCRIPTION
**Pull request by anuraghazra**

### CODE REVIEW

File: packages/blade/src/components/Typography/BaseText/BaseText.stories.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import type { BaseTextProps } from './BaseText';
 import BaseTextComponent from './BaseText';
 
 export default {
-  title: 'Components/Typography/BaseText',
+  title: 'Components/Typography/BaseText (Internal)',
```
Comment (kamleshchandnani) [inline packages/blade/src/components/Typography/BaseText/BaseText.stories.tsx:8]: can you rebase with master and make these changes for other components like Button that we shipped recently?

Comment (anuraghazra) [reply packages/blade/src/components/Typography/BaseText/BaseText.stories.tsx:8]: Will do

Comment (anuraghazra) [reply packages/blade/src/components/Typography/BaseText/BaseText.stories.tsx:8]: @kamleshchandnani button already has it, BaseText too. We don't have any other internal components I think. 

We can merge this if you want.

**Comment by kamleshchandnani** [comment]
@chaitanyadeorukhkar @anuraghazra is there anything pending on this? I saw Chaitanya was saying it's not working for him. Are things clear on that?

**Comment by anuraghazra** [comment]
@kamleshchandnani let's close this I'll fix the other bug in different PR. (:p this one is pending for long)

**Comment by kamleshchandnani** [comment]
what is the bug though? i can see this toggle working fine on this preview link?

**Comment by anuraghazra** [comment]
> what is the bug though? i can see this toggle working fine on this preview link?

The toggling works, the bug is in the "docs" panel. 


See in here the [examples are not loading. ](https://61c19ee8d3d282003ac1d81c-setjbdlkti.chromatic.com/?path=/docs/components-button-basebutton-internal--base-button)

Screenshot 2022-07-19 at 11 42 01 AM

**Pull request merged**